contactsummary/js/src/contact-summary/app.js

```diff
@@ -2,22 +2,23 @@
 
 define([
   'common/angular',
+  'leave-absences/shared/components/leave-widget/leave-widget.component',
   'contact-summary/modules/filters',
   'contact-summary/modules/services',
   'contact-summary/modules/settings',
   'contact-summary/controllers/contactSummary',
   'contact-summary/controllers/keyDates',
   'contact-summary/controllers/keyDetails',
-  'contact-summary/components/leave-widget/leave-widget.component',
   'contact-summary/directives/donutChart'
 ], function (angular) {
   var app = angular.module('contactsummary', [
     'ngRoute',
     'ngResource',
     'ui.bootstrap',
     'common.services',
+    'leave-absences.components',
+    'leave-absences.settings',
```

Comment:
i think contacsummary shouldn't know anything about ` leave-absences.settings`, this looks like an exposure of internal details of L&A.  `leave-absences.components`, should be the one defining as a dependency `leave-absences.settings`